pandas/core/reshape/merge.py

```diff
@@ -2346,3 +2375,37 @@ def renamer(x, suffix):
         )
 
     return llabels, rlabels
+
+
+def _chunks(lst, n):
+    for i in range(0, len(lst), n):
+        yield lst[i : i + n]
+
+
+# TODO: perform lazy merge as optimized cython, rather than chunked merge
```

Comment:
do we have an issue for this?

Reply:
no issue set up yet, since there's nothing in master that this issue could be associated with (can we open issues for code still in PR)